frames/src/main/java/com/checkout/frames/screen/billingaddress/billingaddressdetails/models/BillingFormFields.kt

```diff
@@ -13,7 +13,7 @@ package com.checkout.frames.screen.billingaddress.billingaddressdetails.models
  *     Phone - Mandatory and can not be optional field
  */
 public enum class BillingFormFields(internal var isFieldOptional: Boolean = false) {
-    FullName,
+    CardHolderName,
```

Comment:
The comment to be updated in line 6

Reply:
There are some FullName references in the `/docs`, shall we update them?

Reply:
Of course yes. However, updating a doc is the final step along with the release process of Frames SDK when we create a release PR. Refer [release process](https://checkout.atlassian.net/wiki/spaces/FS/pages/1476133559/Mobile+Frames+SDK+-+Release+Process#:~:text=Create%20a%20release%20staging%20branch%20named%C2%A0release/RELEASE_ticketnumber_x.x.x%20from%20the%20master%20branch)